sycl/test/basic_tests/stream/blocking_pipes_and_stream.cpp

```diff
@@ -0,0 +1,81 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out %ACC_CHECK_PLACEHOLDER
```

Comment:
Does this test make sense without ACC available (i.e. compile only mode)?
If not, then it would be useful to add a REQUIRE statement here.

Reply:
I believe this test should still run on any available device:
 - on accelerator: in order to test this feature works on accelerator;
 - on gpu/cpu: in order to test this feature doesn't break anything (if `pipe` class is applicable for cpu/gpu).

Reply:
Pipes are applicable only for FPGA, that's why added REQUIRE statement.